vector/src/main/java/im/vector/util/MediaAVScanner.java

```diff
@@ -16,36 +16,20 @@
 
 package im.vector.util;
 
-import android.content.Context;
 import android.support.annotation.Nullable;
 import android.text.TextUtils;
-
+import android.util.Log;
 import org.matrix.androidsdk.rest.callback.ApiCallback;
 import org.matrix.androidsdk.rest.model.crypto.EncryptedFileInfo;
-import org.matrix.androidsdk.util.ContentUtils;
-import org.matrix.androidsdk.util.Log;
-
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
-import java.io.ObjectInputStream;
-import java.io.ObjectOutputStream;
-import java.util.HashMap;
+import im.vector.activity.RiotAppCompatActivity;
+import io.realm.Realm;
+import io.realm.RealmResults;
 
-public class MediaAVScanner {
+public class MediaAVScanner extends RiotAppCompatActivity {
```

Comment:
MediaAVScanner is not an Activity, you cannot do that. Instead, pass realm instance as a parameter for methods who need it